src/content/getting-started/create-workload-cluster/index.md

```diff
@@ -4,239 +4,315 @@ title: Creating a workload cluster and installing applications
 description: The Giant Swarm Management API is easily accessible by means of kubectl-gs, a kubectl plugin developed by Giant Swarm. This tutorial guides you throughout some of the most important features of the tool.
 weight: 40
 aliases:
+  - /getting-started/capa-wlcluster-creation/
+  - /getting-started/capz-cluster-creation/
   - /getting-started/kubectl-gs/
 menu:
   main:
     parent: getting-started
 owner:
   - https://github.com/orgs/giantswarm/teams/team-honeybadger
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
 user_questions:
-  - How can I access the Giant Swarm Management API?
   - How do I use kubectl gs?
-last_review_date: 2022-02-28
+  - How can I create a workload cluster?
+last_review_date: 2023-06-12
 ---
 
-As better explained in the [reference]({{< relref "/use-the-api/kubectl-gs" >}}), kubectl-gs is a CLI and a kubectl plugin (invoked as kubectl gs) for the Giant Swarm Management API. In this guide, you will learn how to use `kubectl-gs` for your daily tasks on the Giant Swarm installation.
+To run your business applications on Kubernetes, you need a workload cluster. The `kubectl-gs` tool ([reference]({{< relref "/use-the-api/kubectl-gs" >}})), as already installed in the previous tutorial, is used to create such clusters.
 
-## Step 1: Installing the necessary tools
+## Step 1: Log in to the management cluster
 
-In order to use `kubectl-gs`, you need to have a few tools installed in your computer:
+Please follow the article [Access to the management cluster]({{< relref "/getting-started/management-cluster" >}}). In short, you need to point kubectl to the management cluster:
 
-1. install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
-2. install [krew](https://krew.sigs.k8s.io/)
-3. install `kubectl-gs` by running: `kubectl krew install gs`
+```sh
+kubectl gs login "https://api.<management cluster domain>/" --cluster-admin
```

Comment:
is `--cluster-admin` needed/allowed for customers? sounds like a GS feature but I might very well be wrong

Reply:
You're right. The flag must be removed so that the customer's OIDC provider(s) are shown on the web page for authentication.